test/unit/sdk.test.js

```diff
@@ -140,7 +144,7 @@ describe('sdk', () => {
           isSmartcarHosted: false,
           code: 'super-secret-code',
           error: undefined,
-          state: 'some-state',
+          state: window.btoa(`{"instanceId":"${smartcar.instanceId}","originalState":"some-state"}`),
```

Comment:
Can we use the `getEncodedState()` function for these? If we need to customize the state we can make the helper have a default value:
```js
const getEncodedState = (instanceId, originalState = originalState) => ...
```

Reply:
Good idea! Much cleaner.